policyengine_core/parameters/parameter_node.py

```diff
@@ -274,3 +276,11 @@ def get_child(self, path: str) -> "ParameterNode":
                     f"Could not find the parameter (failed at {name})."
                 )
         return node
+
+    def write_yaml(self, file_path: Path) -> yaml:
+        data = self.get_attr_dict()
```

Comment:
It's at this point where I'd do some of what you've done in `AtInstantLike`, though I'm not sure you need to hard-code the keys we need. I would just define a list of the keys we don't want, run the function from here, use the list to remove any keys we don't want, then recurse here, as you've done